app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -3,14 +3,20 @@
 class CoronavirusLandingPageController < ApplicationController
   def show
     @statistics = FetchCoronavirusStatisticsService.call
+    @timeline_nations_items = timeline_nations_items
     if @statistics
       set_expiry 5.minutes
     else
       logger.warn "Serving /coronavirus without statistics"
       set_expiry 30.seconds
     end
 
-    @content_item = content_item.to_hash
+    @content_item = if params[:timeline_nation] && !Rails.env.production?
+                      timeline_nations_content_item.to_hash
```

Comment:
It appears that brakeman doesn't trust models. Calling `CoronavirusTimelineNationsContentItem.load.to_hash` directly in the `if` caused brakeman to throw a security warning:

== Warning Types ==

Cross-Site Scripting: 1

== Obsolete Ignore Entries ==

344ea763c233c1799950725baf2a9cd55fca8ccb96a0e67af99fbaee080b4487
840e07ce0072ce08b8e6875bde330c91ba40ac3cd6545a95952a567008a3aa83
fcdb8d748855278e7cb8d9639a2f0a4fd446d022d744162a440d2d967d0a2ae3

== Warnings ==

Confidence: Weak
Category: Cross-Site Scripting
Check: CrossSiteScripting
Message: Unescaped model attribute
Code: JSON.pretty_generate(presenter.faq_schema((CoronavirusTimelineNationsContentItem.load.to_hash or content_item.to_hash)))
File: app/views/coronavirus_landing_page/components/shared/_schema.html.erb
Line: 2

I had a look at some [docs](https://brakemanscanner.org/docs/warning_types/cross_site_scripting/) and rather than ignore the warning, I'm just going to keep it wrapped in the method.